modules/sdk-coin-zeta/package.json

```diff
@@ -46,7 +46,8 @@
     "@cosmjs/amino": "^0.29.5",
     "@cosmjs/encoding": "^0.29.5",
     "@cosmjs/stargate": "^0.29.5",
-    "bignumber.js": "^9.1.1"
+    "bignumber.js": "^9.1.1",
+    "@bitgo/sdk-lib-mpc": "^8.5.0"
```

Comment:
@Vijay-Jagannathan master branch is currently broken https://github.com/BitGo/BitGoJS/actions/runs/6015424750/job/16317283340

Reply:
weird how it didn't show up in the [build](https://github.com/BitGo/BitGoJS/actions/runs/5998299142/job/16266306825?pr=3835). Also shouldn't it work for all minor versions ?

Reply:
Being addressed in this PR - https://github.com/BitGo/BitGoJS/pull/3859